_docs_integrate/attribute-introduction.md

```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
-The sharing of a RelationshipAttribute by its `owner` with such a peer would in turn lead to the creation of another own shared RelationshipAttribute, which contains the Address of this peer within its `shareInfo.peer` property.
-However, as this own shared RelationshipAttribute originates from another own shared RelationshipAttribute, it contains the `id` of the source own shared RelationshipAttribute within its `shareInfo.sourceAttribute` property.
+The own shared RelationshipAttribute is the LocalAttribute of the `owner` of the RelationshipAttribute and the peer's LocalAttribute is referred to as a peer shared RelationshipAttribute.
 
-Sometimes an own shared RelationshipAttribute is referred to as an own shared ThirdPartyRelationshipAttribute. This is the case if it originates from another own shared RelationshipAttribute that exists in the context of a different Relationship. An own shared ThirdPartyRelationshipAttribute can be identified by the fact that the `id` of the own shared RelationshipAttribute used as the source is stored in its `shareInfo.sourceAttribute` property. This is the case at least until the own shared RelationshipAttribute used as the source has not been [deleted]({% link _docs_integrate/delete-attributes.md %}#delete-own-shared-attributes).
-{: .notice--info}
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/dd79b31c-0404-48bb-9773-9b989801c03c" id="RbDyLSjh7BzY"></iframe></div>
 
-#### Peer shared RelationshipAttributes
+Of course, the `shareInfo` property of the own shared RelationshipAttribute is set. Within the `shareInfo.peer` property, the Address of the peer to whom the RelationshipAttribute's `owner` has established the Relationship is specified.
+If the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, the `shareInfo.sourceAttribute` property is undefined.
+The peer shared RelationshipAttribute is the peer's [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) and forms the counterpart of the own shared RelationshipAttribute. The `id` of a peer shared RelationshipAttribute is always the same as the `id` of the associated own shared RelationshipAttribute. As with the own shared RelationshipAttribute, the `shareInfo` property of a peer shared RelationshipAttribute is of course set.
+Within the `shareInfo.peer` property, the Address of the `owner` of the RelationshipAttribute is specified.
+It should be emphasized that this is characteristic for a peer shared RelationshipAttribute.
```

Comment:
When I wrote this comment, I was thinking about the difference between peer shared and third party owned RelationshipAttributes. But now I noticed that it is the same for peer shared RelationshipAttributes and peer shared ThirdPartyRelationshipAttributes. With the strict division between them now, this sentence could be confusing as it might suggest a difference between peer shared RelationshipAttributes and peer shared ThirdPartyRelationshipAttributes which would be wrong. So, I'd kindly ask you to remove it again. 🙈

Reply:
No problem 👯‍♀️